modules/database/export.bal

```diff
@@ -281,7 +281,7 @@ public isolated transactional function getTimelineStepLimit(int experimentId) re
     // check if < 10000 steps
     int count = check experimentDB->queryRow(sql:queryConcat(
         `SELECT count(*) FROM TimelineStep `,
-        timelineStepListFilter(experimentId, (), (), (), ()), `;`
+        timelineStepListFilter(experimentId, (), (), (), (), ()), `;`
```

Comment:
Is this required or can optional parameters be left out like they can in javascript? (can be left as is, just looks weird...)

Reply:
Optional parameters (i.e. `int? x`) can't be left out. However, if a default value is specified (i.e. `int? x = ()`), we can avoid all the `nil` parameters in the function call.